.gitignore

```diff
@@ -2,17 +2,14 @@
 
```

Comment:
please remove .* and all ! patterns and just add ignoring files one by one folder/file so we have a clean file